conda-recipe/rsmtool/meta.yaml

```diff
@@ -19,55 +21,19 @@ build:
     - render_notebook = rsmtool.reporter:main
     - convert_feature_json = rsmtool.convert_feature_json:main
 
+{% block requirements -%}
 requirements:
   build:
-    - python
-    - numpy>=1.14.0,<1.15
-    - scipy>=1.1.0,<1.2.0
-    - ipython>=6.5.0,<6.5.1
-    - jupyter>=1.0.0,<1.1
-    - joblib>=0.11,<0.12
-    - matplotlib>=2.1.2,<2.2
-    - nose>=1.3.7,<1.4
-    - notebook>=5.7.2,<5.8
-    - pandas>=0.23.4,<0.23.5
-    - scikit-learn>=0.19.1,<0.19.2
-    - seaborn>=0.9.0,<0.10.0
-    - skll>=1.5,<1.6
-    - statsmodels>=0.9.0,<0.9.1
-    - sphinx
-    - sphinx_rtd_theme
-    - zeromq
-    - coverage
-    - parameterized
-    - setuptools
-    - openpyxl
-    - xlrd
-    - xlwt
-
+    - python 3.6.8
```

Comment:
I think it might be nice to build `noarch` packages that can still pin Python versions. For example, see [here](https://github.com/JRCSTU/jrcstu-forge/commit/12b0695738ece2e5d03e16e319de81a4be62ead5). Also, I don't think we want to pin to a specific version of Python 3.6. Any Python >= 3.6 should work fine, including Python 3.7 right?

Reply:
I agree with everything, except I wasn't sure about 3.7. I've never tested anything with 3.7, and the old `conda_requirements.txt` had `python>=3.6.3,<3.7`.